oe_authentication.install

```diff
@@ -18,6 +18,7 @@ function oe_authentication_install() {
   \Drupal::configFactory()->getEditable('cas.settings')
     ->set('server.hostname', 'ecas.ec.europa.eu')
     ->set('server.path', '/cas')
+    ->set('server.version', '3')
```

Comment:
Why are we setting version number 3 here? The default configuration the module installs with is for ECAS. In the runner we override these configuration to make it work on our dev envs with PCAS.

Reply:
Hello, I tested with both and it works.
For more consistency with the rest, I would leave it like it is.